### PR TITLE
Bps 798 read blob from configured container config changes

### PR DESCRIPTION
**Pull request by mdayican**
JIRA link (if applicable)
https://tools.hmcts.net/jira/browse/BPS-798

Change description

configuration changes
read blob from configured container , "ALL" means read from all available containers .

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[x] No

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -127,6 +127,8 @@ module "bulk-scan" {
     STORAGE_BLOB_LEASE_TIMEOUT               = "${var.blob_lease_timeout}"               // In seconds
     STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES = "${var.blob_processing_delay_in_minutes}"
 
+    STORAGE_BLOB_SELECTED_CONTAINER ="${var.blob_selected_container}"
```
Comment (doncem) [inline infrastructure/main.tf]: missing space after `=`

File: infrastructure/variables.tf [hunk 2]
```diff
@@ -172,6 +172,11 @@ variable "blob_processing_delay_in_minutes" {
   default = "0"
 }
 
+variable "blob_selected_container" {
+  default = "ALL"
+}
+
+
```
Comment (doncem) [inline infrastructure/variables.tf]: ocd: extra empty line not needed

File: infrastructure/main.tf (hunk 1 above)
Comment (mdayican) [reply infrastructure/main.tf]: fixed

> missing space after `=`

fixed

File: infrastructure/variables.tf (hunk 2 above)
Comment (mdayican) [reply infrastructure/variables.tf]: > ocd: extra empty line not needed

fixed

**Pull request merged**